soundata/datasets/fsdnoisy18k.py

```diff
@@ -0,0 +1,319 @@
+            metadata,
+        )
+
+        self.audio_path = self.get_path("audio")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The clip's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @property
+    def tags(self):
+        return annotations.Tags([self._clip_metadata.get("tag")], np.array([1.0]))
+
+    @property
+    def aso_id(self):
+        return self._clip_metadata.get("aso_id")
+
+    @property
+    def manually_verified(self):
+        return self._clip_metadata.get("manually_verified")
+
+    @property
+    def noisy_small(self):
+        return self._clip_metadata.get("noisy_small")
+
+    @property
+    def split(self):
+        return self._clip_metadata.get("split")
+
+    def to_jams(self):
+        """Get the clip's data in jams format
+
+        Returns:
+            jams.JAMS: the clip's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path, tags=self.tags, metadata=self._clip_metadata
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO, sr=None) -> Tuple[np.ndarray, float]:
+    """Load a FSDnoisy18K audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+        sr (int or None): sample rate for loaded audio, 44100 Hz by default.
+            If different from file's sample rate it will be resampled on load.
+            Use None to load the file using its original sample rate (sample rate
+            varies from file to file).
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    audio, sr = librosa.load(fhandle, sr=sr, mono=True)
+    return audio, sr
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """
+    The FSDnoisy18K dataset
+    """
+
+    def __init__(self, data_home=None):
+        super().__init__(
+            data_home,
+            name="fsdnoisy18k",
+            clip_class=Clip,
+            bibtex=BIBTEX,
+            remotes=REMOTES,
+            license_info=LICENSE_INFO,
+        )
+
+    @core.copy_docs(load_audio)
+    def load_audio(self, *args, **kwargs):
+        return load_audio(*args, **kwargs)
+
+    @core.cached_property
+    def _metadata(self):
+
+        metadata_train_path = os.path.join(
+            self.data_home, "FSDnoisy18k.meta", "train.csv"
+        )
+        metadata_test_path = os.path.join(
+            self.data_home, "FSDnoisy18k.meta", "test.csv"
+        )
+
+        if not os.path.exists(metadata_train_path):
+            raise FileNotFoundError(
+                "Train metadata not found. Did you run .download()?"
+            )
+        if not os.path.exists(metadata_test_path):
+            raise FileNotFoundError("Test metadata not found. Did you run .download()?")
```

Comment:
Just this line is not covered by tests, let's just fix that and ready to merge! <3